_includes/tutorials/message-ordering/kafka/markup/dev/make-gradle-wrapper.adoc

```diff
@@ -0,0 +1,5 @@
+Run the following command to obtain the Gradle wrapper:
```

Comment:
Should a note be added to this tutorial that the user needs to have Gradle installed?

Reply:
Many tutorials use Gradle and I don't think any mention it as a pre-req. In fact, I don't think any KT identifies required local installs to run (in contrast the README mentions pre-reqs for local testing).  Maybe we file a GH issue to track separately?

Reply:
A separate issue sounds fine to me if any changes are needed at all here.

Reply:
Filed https://github.com/confluentinc/kafka-tutorials/issues/591 for tracking